plugins/obs-outputs/CMakeLists.txt

```diff
@@ -65,6 +69,7 @@ target_link_libraries(
   PRIVATE OBS::libobs
           OBS::happy-eyeballs
           OBS::opts-parser
+          OBS::bpm
```

Comment:
```suggestion
```

Reply:
Thanks @tytan652. I just force pushed an update that moves this to the UI/CMakeLists.txt file.